### PR TITLE
fish: update highlight queries

### PR DESCRIPTION
**Pull request by taekwombo**
I'm dropping this in case upcoming [fish breaking changes](https://github.com/ram02z/tree-sitter-fish/pull/16) are merged.

### CODE REVIEW

**Comment by clason** [comment]
Thanks! It would be helpful if you updated the parser while you're at it (just replace the commit id in the `lockfile.json`) -- this will prevent the upcoming automatic update from breaking and blocking other updates.

**Comment by clason** [comment]
Ah, I see that your parser PR isn't merged yet. I'll mark this as draft to indicate it shouldn't be merged yet. It'd be best if you added the parser bump yourself here when the parser PR is merged.

File: queries/fish/highlights.scm [hunk 1]
```diff
@@ -7,15 +7,22 @@
...
 
+
+; match operators of "test" command
+(command
+  name: (word) @function (#match? @function "^test$")
+  argument: (word) @operator (#match? @operator "^(!?=|-[a-zA-Z]+)$"))
+
+; match operators of "["
+(test_command
+  argument: (word) @operator (#match? @operator "^(!?=|-[a-zA-Z]+)$"))
```
Comment (theHamsta) [inline queries/fish/highlights.scm:24]: `\\=` works

**Comment by theHamsta** [comment]
Merged with https://github.com/nvim-treesitter/nvim-treesitter/pull/3851

**Pull request closed without merging**